app.js

```diff
@@ -14,12 +15,25 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers().then(beersFromApi => {
+    console.log(beersFromApi);
+    res.render('beers', { beers: beersFromApi });
+  });
+});
+
+app.get('/random-view', (req, res) => {
+  punkAPI.getRandom().then(beersFromApi => {
+    res.render('random-view', { randomBeer: beersFromApi });
```

Comment:
You can consider extracting the object (random beer) from the array, to have a cleaner code but it works perfectly also this way ;)